### PR TITLE
Reduce the number of jquery and bump

### PR DESCRIPTION
**Pull request by Preovaleo**
solve  #2067

### CODE REVIEW

**Comment by ozh** [comment]
Does jQuery 2.x break things ?

**Comment by Preovaleo** [comment]
I don't know but yes it could be a good idee

**Comment by Preovaleo** [comment]
yes it's working

File: .gitignore [hunk 1]
```diff
@@ -50,3 +50,6 @@ Desktop.ini
 
 # Mac crap
 .DS_Store
+
+# NetBeans files
+/nbproject/
```
Comment (GaryJones) [inline .gitignore]: This IDE / OS specific stuff should be in your own global gitignore, not at the project level.

**Pull request closed without merging**